src/routes/AuthInfo.jsx

```diff
@@ -0,0 +1,34 @@
+import { onAuthStateChanged, signOut } from 'firebase/auth';
+import React, { useEffect, useState } from 'react';
+import { auth } from '../utils/firebase';
+import SignOutButton from '../components/SignOutButton';
+
+const AuthInfo = () => {
+  const [authUser, setAuthUser] = useState(null);
+  useEffect(() => {
+    const listen = onAuthStateChanged(auth, (user) => {
```

Comment:
we are already calling this on Login.jsx and it's automatically called once the auth state has changed, so we don't need to use it inside useEffect..

Reply:
just delete this whole component

Reply:
also, can you put onAuthStateChanged in Login.jsx in useEffect? Since it's not in useEffect, the function is called infinitely.

Reply:
^I will just push do this one right now and push it to main. this will slow down the app for everyone